pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -136,6 +180,16 @@ let
       DAMON_DBGFS = whenAtLeast "5.15" yes;
       DAMON_RECLAIM = whenAtLeast "5.16" yes;
       DAMON_LRU_SORT = whenAtLeast "6.0" yes;
+      # Support recovering from memory failures on systems with ECC and MCA recovery.
+      MEMORY_FAILURE = yes;
+
+      # Collect ECC errors and retire pages that fail too often
+      RAS_CEC                   = yes;
+    } // optionalAttrs (stdenv.is32bit) {
+      # Enable access to the full memory range (aka PAE) on 32-bit architectures
+      # This check isn't super accurate but it's close enough
+      HIGHMEM                   = option yes;
+      BOUNCE                    = option yes;
```

Comment:
Do these even do anything? At least `BOUNCE` seems to be upstream default anyway if I understand the kconfig files correctly.

Reply:
They're enabled on x86-32 and armv7l, but I'm less confident about the other arches, and it's probably worth keeping them listed explicitly just to make sure they never get disabled quietly.